grouper/constants.py

```diff
@@ -71,3 +71,6 @@
 
 # Maximum length a name can be. This applies to user names and permission arguments.
 MAX_NAME_LENGTH = 128
+
+# Grouper used UserMetadata data_keys
+SHELL_MD_KEY = "shell"
```

Comment:
Got confused when first reading this variable name. Can we be more explicit? Something like `USER_METADATA_SHELL_KEY`. other keys will keep the same pre- and suf- fix.